app/src/organisms/QuickTransferFlow/QuickTransferAdvancedSettings/PipettePath.tsx

```diff
@@ -27,6 +33,16 @@ interface PipettePathProps {
 export function PipettePath(props: PipettePathProps): JSX.Element {
   const { onBack, state, dispatch } = props
   const { t } = useTranslation('quick_transfer')
+  const keyboardRef = React.useRef(null)
+
+  const [selectedOption, setSelectedOption] = React.useState<PathOption>(
+    state.path
+  )
+  const [currentStep, setCurrentStep] = React.useState<number>(0)
+  const [blowOutLocation, setBlowOutLocation] = React.useState<string>(
+    state.blowOut ?? 'trashBin'
+  )
+  const [volume, setVolume] = React.useState<number>(state.volume)
 
   const allowedPipettePathOptions: PathOption[] = ['single']
   if (state.sourceWells.length === 1 && state.destinationWells.length > 1) {
```

Comment:
we also need to take in consideration the volume here -- I think this logic should be:

```suggestion
    if (
    state.transferType === 'consolidate' &&
    volumeLimits.max >= state.volume * 2
  ) {
    allowedPipettePathOptions.push('multiDispense')
  } else if (
    state.transferType === 'distribute' &&
    volumeLimits.max >= state.volume * 2
  ) {
    allowedPipettePathOptions.push('multiAspirate')
  }
```